dali/kernels/imgproc/convolution/laplacian_cpu.h

```diff
@@ -0,0 +1,311 @@
+// Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_KERNELS_IMGPROC_CONVOLUTION_LAPLACIAN_CPU_H_
+#define DALI_KERNELS_IMGPROC_CONVOLUTION_LAPLACIAN_CPU_H_
+
+#include <memory>
+
+#include "dali/core/tensor_view.h"
+#include "dali/kernels/common/utils.h"
+#include "dali/kernels/imgproc/convolution/separable_convolution_cpu.h"
+#include "dali/kernels/imgproc/convolution/convolution_cpu.h"
+#include "dali/kernels/kernel.h"
+#include "dali/kernels/scratch.h"
+#include "dali/pipeline/util/operator_impl_utils.h"
+
+namespace dali {
+namespace kernels {
+
+namespace laplacian {
+
+using namespace conv_transform;  // NOLINT
```

Comment:
Do we need this here? I think I saw only a few references to this namespaces, maybe it would be cleaner to do it explicitly?